lib/puppet/provider/mount/parsed.rb

```diff
@@ -23,7 +23,7 @@
   when "Solaris"
     @fields = [:device, :blockdevice, :name, :fstype, :pass, :atboot, :options]
   else
-    @fields = [:device, :name, :fstype, :options, :dump, :pass]
+    @fields = [:device, :name, :fstype, :options, :dump, :pass, :live_options]
```

Comment:
My guess is that the reason our output is broken is because `parsedfile` is actually trying to append the new `live_options` field. I think you want to omit this from the fields variable. We may need to find a way to trick `parsedfile` into accepting the extra data and passing it into the `property_hash` once we remove this, though